packages/cardano-services/src/Projection/createTypeormProjection.ts

```diff
@@ -118,6 +118,7 @@ export const createTypeormProjection = ({
   });
   return projectionSource$.pipe(
     applyMappers(mappers),
+    CardanoNodeUtil.bufferChainSyncEvent<ProjectionEvent>(10),
```

Comment:
This should be inferred from source$
```suggestion
    CardanoNodeUtil.bufferChainSyncEvent(10),
```

Reply:
fd7ef848c and 3aeb79ca1 should solve this